### PR TITLE
Form validator feature

### PR DESCRIPTION
**Pull request by Rebelzob**
# Portfolio

- Added Javascript validator for lowercase email verification.
- Added CSS styles.
- HTML form re-coded.

**Please take into consideration that I don't have a coding partner this week so I don't have commits from anybody else in my project from this week, here is the [link](https://github.com/Rebelzob/Portfolio/pull/13) to the last PR with the support**

- Letter proving the reason for working alone.
image

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @Rebelzob,
 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the student's success team and they'll permit you to work solo or look for a suitable partner. Then add a screenshot of the response from student success asking you to work alone.

## Invalid Code Review Request🛑 

- [ ] Kindly update your linters.yml file for this project to include ESLinters. In the root project, you have a folder named .github. Inside this one, you have another folder named workflows. And inside the workflows folder, you have a file named linters.yml. Inside this file, you need to update the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js);

| Desp  | Images |
| ------------- | ------------- |
| Actions without ESLint  | image  |
| Actions with ESLint  | image  |

> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
  
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Rebelzob** [comment]
- Here is the letter of Microverse telling me to work alone

image

**Comment by topeogunleye** [comment]
> * Here is the letter of Microverse telling me to work alone
> 
> image

@Rebelzob, Kindly edit your pull request description and put this picture there

image

File: index.html [hunk 1]
```diff
@@ -182,17 +182,18 @@ <h4 class="ab-titles skills"> Skills</h4>
       <div class="infography-contact">
         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
       </div>
-      <div class="form flex-column">
-        <form class="flex-column " action="https://formspree.io/f/mayzgdra" method="POST" class="form-data">
```
Comment (Alusp) [inline index.html:186]: - [ ] Good job in implementing the following changes, :clap: Your Email Validation works successfully, However, Your form is not submitting the form using formspree, Please ensure that your form submit successfully when it passes the validation checks.

File: index.html [hunk 2]
```diff
@@ -182,17 +182,18 @@ <h4 class="ab-titles skills"> Skills</h4>
...
+      <div class="flex-column">
+        <form id="form" class="flex-column form-data" action="'https://formspree.io/f/mayzgdra'" method="POST">
           <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
-          <div class="first-last-name">
+          <!-- <div class="first-last-name">
             <input type="text" name="first-name" id="first-name" placeholder="First Name" required maxlength="30">
             <input type="text" name="last-name" id="last-name" placeholder="Last Name" required maxlength="30">
-          </div>
+          </div> -->
```
Comment (Alusp) [inline index.html:191]: - [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Rebelzob, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close

## Highlights
- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @Rebelzob 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Removed commented-out code ✔️ 
Form submitted successfully ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**